plugins/environment.go

```diff
@@ -115,11 +116,17 @@ When the -plugin option is specified, these flags are ignored.`)
 		err = proto.Unmarshal(apiData, documentv3)
 		if err == nil {
 			env.Request.AddModel("openapi.v3.Document", documentv3)
+			sourceName := strings.ReplaceAll(*input, ".pb", ".yaml")
+			if _, err := os.Stat(sourceName); os.IsNotExist(err) {
+				// sourceName does not exist exist. Lets try .json instead
```

Comment:
what if the `.json` version doesn't exist? I'm assuming it will fail elsewhere? Might be worth it to fail here.

Reply:
Done